### PR TITLE
Gosling-fix

### PR DESCRIPTION
**Pull request by igler**
Fixes "No suitable constructor found..." issues when upgrading to Gosling release train.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 	<groupId>org.springframework.data</groupId>
 	<artifactId>spring-data-envers</artifactId>
-	<version>0.3.0.BUILD-SNAPSHOT</version>
```
Comment (olivergierke) [inline pom.xml]: No need to upgrade the version number manually as we do this after a release. 0.3.0.BUILD-SNAPSHOT is the correct version. Please remove from PR.

File: pom.xml [hunk 2]
```diff
@@ -58,8 +58,8 @@
 	</scm>
 
 	<properties>
-		<spring.version>4.0.9.RELEASE</spring.version>
-		<spring.data.jpa.version>1.8.0.RELEASE</spring.data.jpa.version>
+		<spring.version>4.2.1.RELEASE</spring.version>
```
Comment (olivergierke) [inline pom.xml]: Please don't raise the Spring dependency as we usually do this in a controlled, dedicated way. There might be a need to upgrade to Spring 4.1 but let's try to not artifically cut off users by requiring the most recent version. Users are still free to use the most recent one in their apps though.

**Comment by olivergierke** [comment]
Thanks for that! Would you mind tweaking the PR according to my comments? Happy to do then release 0.3.

**Comment by igler** [comment]
OK, "updated" pom.xml according to your comments.

**Comment by olivergierke** [comment]
That's merged.

**Pull request closed without merging**